ext/nnstreamer/tensor_filter/tensor_filter_snpe.cc

```diff
@@ -184,6 +204,22 @@ const char * snpe_subplugin::runtimeToString (zdl::DlSystem::Runtime_t runtime)
   }
 }
 
+/**
+ * @brief Internal method to get names of output layers from tensors information.
+ */
+bool snpe_subplugin::set_output_layer_names (const GstTensorsInfo *info)
+{
+  for (unsigned int i = 0; i < info->num_tensors; ++i) {
+    if (info->info[i].name == nullptr) {
+      /* failed */
```

Comment:
let's consider both 'custom' and 'outputname'.
info[i].name is null, clear `output_layer_names` and return (Using single-shot Java API always returns failure.)

Reply:
I replace the type of output_layer_names to `std::vector<std::string>`, because the `zdl::DlSystem::StringList` does not open `clear` method.
Please check!